module/Toolbar.kt

```diff
@@ -236,7 +236,7 @@ object Toolbar {
                         height = TOOLBAR_BUTTON_SIZE,
                         buttons = listOf(
                             TextButtonArg(
-                                text = schema.name.lowercase(),
+                                text = Label.SCHEMA.lowercase(),
```

Comment:
We were using the name provided by `typedb-client-java`, but this is part of our interface so we've added appropriate labels to Label.kt.